src/drivers/amd/rembrandt/acp_dmic_dai.c

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+//Copyright(c) 2022 AMD. All rights reserved.
+//
+// Author:	Basavaraj Hiregoudar <basavaraj.hiregoudar@amd.com>
+//		Bala Kishore <balakishore.pati@amd.com>
+
+#include <sof/audio/component.h>
+#include <sof/drivers/acp_dai_dma.h>
+#include <sof/drivers/interrupt.h>
+#include <sof/lib/alloc.h>
+#include <sof/lib/dai.h>
+#include <sof/lib/dma.h>
+#include <sof/lib/uuid.h>
+#include <ipc/dai.h>
+#include <ipc/topology.h>
+#include <platform/fw_scratch_mem.h>
+#include <sof/lib/io.h>
+#include <platform/chip_offset_byte.h>
+
+/*0ae40946-dfd2-4140-91-52-0d-d5-a3-ea-ae-81*/
+DECLARE_SOF_UUID("acp_dmic_dai", acp_dmic_dai_uuid, 0x0ae40946, 0xdfd2, 0x4140,
+		0x91, 0x52, 0x0d, 0xd5, 0xa3, 0xea, 0xae, 0x81);
+
+DECLARE_TR_CTX(acp_dmic_dai_tr, SOF_UUID(acp_dmic_dai_uuid), LOG_LEVEL_INFO);
+
+static inline int acp_dmic_dai_set_config(struct dai *dai, struct ipc_config_dai *common_config,
+					  void *spec_config)
+{
+	dai_info(dai, "ACP: acp_dmic_set_config");
```

Comment:
prefixes are unneeded, you will already be prefixed with acp_dmic_dai since that is the trace context.